catroweb.yaml

```diff
@@ -351,6 +351,33 @@ paths:
               schema:
                 $ref: '#/components/schemas/UploadErrorResponse'
 
+  '/projects/recommended/{id}':
```

Comment:
We will change the route to `/project/{id}/recommendation` so it will be under `/project/{id}`.

Reply:
there is still a small typo
![image](https://user-images.githubusercontent.com/40868718/90861089-7caea100-e38b-11ea-862f-59a8f6962e98.png)

Remove  the "s" from projects
however, maybe we should add the "s" to "recommendation". Thinking about it "recommendations" sounds better to me. Implicating the response contains multiple recommendations not just one.